internal/pkg/task/task_test.go

```diff
@@ -0,0 +1,34 @@
+package task
+
+import (
+	"github.com/stretchr/testify/require"
+	"testing"
+	"time"
+)
+
+func TestEarliestStartTime(t *testing.T) {
```

Comment:
Do we still need this now that `earliestStartTime` is not there any more. Also we don't usually have test func for private functions because usually the logics can be covered in the public functions that use them. For example, testing could be easily added to `TestEventsWriter_WriteEventsUntilStopped` since the logic of `earliestStartTime` is quite simple.

Reply:
makes sense! removed the old `task_test.go` file since the only test in it is for `earliestStartTime`